### DISCUSSION TITLE
年度总结-2023

### DISCUSSION THREAD

**Discussion by String10** [String10/KGarden#19] [General]
> 今年终于找到一个合适的地方来放一点自己的碎碎念, 新的一年希望自己多多产出些优质的内容吧. 😄

九月初的时候自己拟了一个本科生活的总结, 当时得到的初步结论是: "或许写前端小工具就是我的归宿呢". 这个结论或许为时过早, 不过看到现在的我依然作为初学者摆弄Vue3和Bootstrap之类的前端工具, 说不定最后还真让我说中了.

## 打卡NV实习
去年冬天刚刚落地的我, 坐在回家的车上接到了NV HR打给我的电话, 询问我是否找到了实习 -- 当时已经在面试的一个多月后了. 想也没想, 我直接答应了HR的邀请, 然后在二月份正式作为实习生加入了NV.

远程实习带给我很大的自由度来调整自己的工作时间 -- 虽然每周打工三天上课两天, 周末时间一般也被作业之类的占据了. 度过充实的一个学期后我选择了去上海线下实习, 体验了下NV的"共享工位"和"食堂文化"🤣. 线下相比线上最大的好处就是可以多认识几个同事, 不像线上只有平时几个会有工作交集的人和你讨论工作上的事情, 线下会有不少友善的同事可以一起吃饭一起聊天.

在上海呆了差不多两个月, 然后就因为房租到期暂时中止实习了.

## 闪击夏令营
参加夏令营是早就做好的决定, 不过对于夏令营的准备是匆匆又忙忙. 因为一开始只是想着自己学校的夏令营不去白不去, 所以复习时间被压缩到只有两天. 草率的复习并没有给我很大的信心, 在得知面试位次的时候更是让我大吃一惊.

作为前五个面试的人之一, 我并没有什么可以借鉴的经验. 不过进去之后发现面试的氛围还不错, 在问答之中我的状态逐渐上升, 毕竟在企业面试中我也是有不少的经验的. 我觉得可能"破罐子破摔"和"上不上都行"的思想实在是给我上了很大的buff.

面试结束当天我就准备前往上海了, 当天晚上在收拾行李的时候接到了面试导师的意向询问, 当时感觉可能有戏, 没想到最后还真的保上了. 因为我的笔试成绩并不高, 所以我觉得面试时的表现应该是给了我很大的加分. 虽然最后没有选择升学, 不过我觉得这次的经历的的确确应该被记录下来成为本次总结的一节.

## 速通大三下学期
上个学期选的课都挺不好对付的: 计算机体系结构, 软件体系结构, 组合数学和大数据处理实验. 要不就是理论硬核, 要不就是实验众多, 甚至还挂着实习的debuff, 还好挺下来了. 每周末不是在写实验就是在debug, 实在是很难有什么大块的时间来休息.

一个学期下来, 我觉得相对来说还是两个体系结构最有实用价值. 组合数学虽然很有美感, 但是天赋实在难以支撑我继续深入, 只好浅尝辄止. 大数据处理实验的内容相对也是一言难尽, 框架老旧, 内容生硬, 虽然相对来说这个实验是比较水的, 但是对Hadoop的陌生属实让很多人望而生畏.

计算机体系结构还是很硬核的, 不过凭着"以后可能会有用"的想法最后坚持到了期末. 总结下来我觉得还是从这门课中认识和了解到了很多底层的架构设计, 对于分层和抽象也有了一些新的认识. 但是最重要的是我在NV面试的时候说我学过计算机体系结构来补一补哈. 🤓

软件体系架构我愿称之为SpringBoot实验集合, 如果我没记错的话最后应该是写了10个实验, 前面大部分是迭代的项目, 后面就变成不同软件结构的尝试了. 这门课让我对SpringBoot这个框架有了一个全面的了解, 也没有落下动手实践的工程, 并且对于"商业软件"这个概念也有了一个比较粗浅的认识: "客户端+服务端"或者以此为基础的变体. 另外, 这门课程还介绍了诸如软件即服务(SaaS)等概念.

## 爽玩大四上学期
这个学期真的是到处爽玩, 见识了很多之前没有体验过的事情:

### 第一次线下观赛
结束实习后我先在学校过渡了一周, 然后前往杭州看了一场亚运会足球小组赛. 同学D和我一起去的. 无论是从上海出发还是从杭州返程都是堪堪赶上, 在车站奔跑的画面反而比足球印象更深刻了.

### 押尾光太郎演奏会
同学D和G约我在南京看押尾光太郎的演奏会, 我想着反正也没看过, 那就去吧. 他们提前几天来到南京, 我们还一起去了牛首山. 到了看演奏会那天甚至结束后去同学T的酒吧打了一宿扑克 ("唬牌"实在是太难了, 老实人震怒, 感觉不如一开始打的"掼蛋"). 第二天简直欲仙欲死, 最后回到宿舍大睡一天.

### 悠闲成都三日游
和同学D和同学Y约好去成都耍耍, 第一天看了熊猫基地, 逛了逛春熙路. 第二天带上在成都上学的表妹去了金沙遗址, 在杜甫草堂和人民公园溜达溜达. 第三天直接摆烂了, 晚上在天府机场过的夜, 我的评价是下回一定好好检查在哪个机场. 总体来说成都美食很多, 但是景点只有熊猫基地能打.

### 哈尔滨太阳岛音乐节
室友J邀请我去哈尔滨看音乐节 -- 甚至露天. 全副武装地看完第一天的节目之后我俩决定第二天先去骑马, 把前几个歌手跳过, 只看最后的几个. 龙姨压台演出确实炸裂, 非常好音乐节, 爱来自二手玫瑰. ![梁龙在哈尔滨就变成了凉龙](https://github.com/String10/KGarden/assets/69832835/0ec4376b-2ed3-4767-b504-4c5f46f02e34)
> 图片来自[梁龙在哈尔滨就变成了凉龙](https://www.bilibili.com/video/BV1yj411H7pw/?share_source=copy_web&vd_source=9b0cdeb82015b13a6d6681907445b964)的评论区.

### 其他活动
其他还有比如: 今年去了上海WePlay游戏展, 薅了一堆小礼品; 夏天找初中同学B去了上海欢乐谷; 回家之后去小学同学Z家小聚之类的, 都是2023美好的回忆. ✌

## 读书&追番
今年因为买了JD Plus会员, 为了不浪费送的图书会员开始看书 (非阅读顺序排列):
1. 明朝那些事儿 (1-6): 好看, 打开JD图书的第一本书, 虽然并不算严谨的历史文献, 但是作为历史小说来说趣味性很高, 而且作者的文笔很有感染力, 对于我这种理科生了解明朝历史也有一定的帮助.
2. 活着: 坐一趟地铁就看完了, 差点没绷住, 我的评价是神作.
3. 白鹿原: 好看, 很有乡土风情, 有种史书味道的宏大叙事感.
4. 沉默的大多数: 王小波的杂文, 看过之后总感觉没记住什么.
5. 哈利波特完整系列: 神作, 小时候没看完最后一部, 长大了奖励自己从头再看一遍. 🤭
6. 小偷家族: 挺感人的, 经典的日本文学.
7. 强风吹拂: 22年夏天看过, 然后一口气熬夜看完了, 然后把动画追完, 23年又看了一遍, 非常优秀的励志作品.
8. 兄弟: 余华的一部经典作品, 有一种魔幻的感觉, 但是有的情节非常能够打动人心, 两兄弟的感情跨越了那个年代, 跨越了阶级, 遗憾的是最后没有跨越疾病.
9. 娱乐至死: 早有耳闻, 前来拜读, 初读感想不大, 毕竟已经生活在这个时代中了.
10. 美丽新世界: *娱乐至死*中频繁出现的作品, 里面对于未来世界的想象很有创意, 和*一九八四*应该是两个完全相反的道路.
11. 我不可能是剑神: 无敌流网文, 一般爽, 但是确实停不下来.

今年感觉其实也没看啥其他的 (非阅读顺序排列):
1. 赛博朋克:边缘行者.
2. 进击的巨人最终话.
3. 非自然死亡.
4. [正在看] Chiikawa.
5. [正在看] 葬送的芙莉莲.
6. [正在看] 狩龙人拉格纳.

## 最后两句
不知不觉写了这么长, 但是感觉还有很多事情可以讲. 如果明年在坚持写东西的话, 应该可以分散在月度或者季度总结中. 👌

最近在做的事情:
1. 准备和同学C南下旅行.
2. 研究前端, 优化毕设界面.

---

居然忘了提一件很重要的事:

## 10天速通驾考

差不多10月份初, 当时在家闲来无聊花了若干大洋报名了附近的一个驾校, 寻思考个驾照消磨一下空闲时间. 谁想没过几天我就回了学校, 驾考的事就不了了之了. 回学校前考完了科目一, 练了两三天倒车入库. 没想到过了一个月回来还能差不多记得要领. 回来之后约了11月中旬的科目二和科目三 (约考试那时我还没上过路). 猛练小一周之后满怀忐忑地前往考场: 第一天科目二完美通过, 第二天科目三小有瑕疵. 考完科目三已经是下午了, 在驾校顺带考完了科目四, 最后带着电子驾驶证直接回家了. 😄

*编辑于 2024/01/06 09:33*

**Comment by Wangxf2023** [comment]
nb，10天！

**Reply by String10** [reply]
当时考的时候可紧张了, 还好最后过了😄
